app/api/__init__.py

```diff
@@ -12,10 +12,13 @@ def create_app(test_config=None):
     mongo = PyMongo(app)
     app.app_context().push()
     app.extensions['mongo'] = mongo
+    app.config["JWT_SECRET_KEY"] = os.getenv("SUPER_SECRET")
```

Comment:
Name it JWT_SECRET_KEY, Not SUPER_SECRET. Go with better practice and conventions